default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub, statik }:
+
+buildGoModule rec {
+    name = "upm";
+    version = "908674b";
```

Comment:
Either we can pass the version in as a parameter, or we can omit it entirely. It's primarily for human consumption.